src/misc/inc/MultiDimensionalIndexing.h

```diff
@@ -22,21 +22,21 @@
 //
 //-----------------------------------------------------------------------el-
 
-#ifndef UQ_INTERPOLATION_SURROGATE_HELPER_H
-#define UQ_INTERPOLATION_SURROGATE_HELPER_H
+#ifndef UQ_MULTI_DIMENSIONAL_INDEXING_H
+#define UQ_MULTI_DIMENSIONAL_INDEXING_H
 
 // C++
 #include <vector>
 
 namespace QUESO
 {
-  class InterpolationSurrogateHelper
+  class MultiDimensionalIndexing
```

Comment:
Crap. I didn't think about that really. I don't think so, though, because that code was only used internally in the `InterpolationSurrogate` stuff. In the example, that class is not used at all.